### PR TITLE
Create content

### PR DESCRIPTION
**Pull request by Jasper-Koops**

### CODE REVIEW

File: wend_foosball/scorekeeper/management/commands/create_content.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+                'team_A_back': User.objects.get(username=players[1]).player,
+                'team_B_front': User.objects.get(username=players[2]).player,
+                'team_B_back': User.objects.get(username=players[3]).player,
+                'team_A_score': randint(0, 10),
+                'team_B_score': randint(0, 10),
+            }
+
+            game = Game.objects.create(**kwargs)
+            created_games.append(game)
+            self.stdout.write('createpd game: {}'.format(game))
```
Comment (syberen) [inline wend_foosball/scorekeeper/management/commands/create_content.py]: Spelling error

File: wend_foosball/scorekeeper/management/commands/create_content.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                player = players[x]
+                cup, created = CupResult.objects.get_or_create(
+                    user=player, tournament=tournament, placement=x + 1
+                )
+                if created:
+                    created_results.append(cup)
+
+        return created_results
+
+    def handle(self, *args, **options):
```
Comment (syberen) [inline wend_foosball/scorekeeper/management/commands/create_content.py]: Are these arguments used?

File: wend_foosball/scorekeeper/management/commands/create_content.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+            players = sample(users, 4)
+
+            # Reasons why I prefer subclassed User models, exhibit A:
+            kwargs = {
+                'team_A_front': User.objects.get(username=players[0]).player,
+                'team_A_back': User.objects.get(username=players[1]).player,
+                'team_B_front': User.objects.get(username=players[2]).player,
+                'team_B_back': User.objects.get(username=players[3]).player,
+                'team_A_score': randint(0, 10),
+                'team_B_score': randint(0, 10),
```
Comment (syberen) [inline wend_foosball/scorekeeper/management/commands/create_content.py]: Should probably be 10 - team_A_score

File: wend_foosball/scorekeeper/management/commands/create_content.py [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+        for name in player_names:
+            try:
+                user = User.objects.create_user(username=name, password='test')
+                created_users.append(user)
+                self.stdout.write('created User: {}'.format(name))
+            except IntegrityError:
+                self.stdout.write(
+                    'User {} already exists! Skipping...'.format(name)
+                )
+        return created_users
```
Comment (syberen) [inline wend_foosball/scorekeeper/management/commands/create_content.py]: Question out of interest: why do you return the list of users?

File: wend_foosball/scorekeeper/management/commands/create_content.py (hunk 2 above)
Comment (Jasper-Koops) [reply wend_foosball/scorekeeper/management/commands/create_content.py]: These functions are invoked by Django manage.py and will fail if those arguments are not included, even if they are not used within the functions themselves.

File: wend_foosball/scorekeeper/management/commands/create_content.py (hunk 4 above)
Comment (Jasper-Koops) [reply wend_foosball/scorekeeper/management/commands/create_content.py]: I wrote these functions to be reusable (which is probably a bit overdone as I image these functions will probably only be used in this command) which meant that these functions probably should hav e a way of returning the objects they generated.

File: wend_foosball/scorekeeper/management/commands/create_content.py (hunk 1 above)
Comment (Jasper-Koops) [reply wend_foosball/scorekeeper/management/commands/create_content.py]: Good catch, fixed it!

**Pull request merged**